pkg/apis/migration/migration.go

```diff
@@ -288,6 +291,10 @@ func (volumeMigration *volumeMigration) GetVolumePath(ctx context.Context, volum
 	if volumePath != "" {
 		return volumePath, nil
 	}
+
+	if !createCRIfNotPresent {
+		return "", nil
```

Comment:
return error with not found instead of empty string volume path.

Reply:
!createCRIfNotPresent && volumePath == "", then return error.

Reply:
Divyen, we are already checking for volumePath above so don't need to add that check again. I'll modify to return the a `NotFoundError`.

Reply:
I'm now returning the ErrNotFound